agg-ast/schema_derivation/src/schema_derivation.rs

```diff
@@ -1294,9 +1301,14 @@ impl DeriveSchema for TaggedOperator {
                 args: vec![(*s.var).clone()],
             }
             .derive_schema(state),
-            TaggedOperator::SQLConvert(_) | TaggedOperator::SQLDivide(_) => {
-                Err(Error::InvalidTaggedOperator(self.clone()))
-            }
+            TaggedOperator::Accumulator(_)
```

Comment:
Moved these to error, as we either a. won't get these from an agg pipeline ( ie LIKE, which is a sql construct) or function, which we can't handle (so we'll treat it as invalid)